JavaScripts/subLength.js

```diff
@@ -0,0 +1,20 @@
+const charCount = (string, char) => Array.from(string)
+    .filter((c) => c === char)
+    .length;
+
+const subLength = (string, character) => {
+    if (charCount(string, character) === 2) {
+        const firstIndex = Array.from(string)
+            .findIndex((element) => element === character);
+        const secondIndex = Array.from(string)
+            .indexOf(character, firstIndex + 1);
+        return (secondIndex - firstIndex) + 1
+    }
+    return 0;
```

Comment:
The other way to do it is have

if (charCount(string, character) !== 2) return 0; 

As the first line in the function, and then you don't need o indent anything else. Basically it's doing argument validation immediately and doing an early return if the arguments aren't valid, so the rest of the function can just treat them as valid if it gets to that point.